gooey/internal/scrollbar.lua

```diff
@@ -81,6 +82,49 @@ function M.vertical(handle_id, bounds_id, action_id, action, fn, refresh_fn)
 	return scrollbar
 end
 
+function M.horizontal(handle_id, bounds_id, action_id, action, fn, refresh_fn)
+	handle_id = core.to_hash(handle_id)
+	bounds_id = core.to_hash(bounds_id)
+	local handle = gui.get_node(handle_id)
+	local bounds = gui.get_node(bounds_id)
+	assert(handle)
+	assert(bounds)
+	local scrollbar = core.instance(handle_id, scrollbars, SCROLLBAR)
+	scrollbar.scroll = scrollbar.scroll or vmath.vector3(0, 0, 0)
+	scrollbar.vertical = false
+
+	local handle_size = gui.get_size(handle)
+	local bounds_size = gui.get_size(bounds)
+
+	scrollbar.enabled = core.is_enabled(handle)
+	scrollbar.node = handle
+	scrollbar.node_id = node_id
```

Comment:
*issue*: Oops, `node_id` is never defined in here! `M.vertical` has the same problem!

@britzl was the original intention to set `scrollbar.node_id = handle_id`?